packages/babel-preset-env/test/fixtures/corejs2/usage-browserslist-config-ignore/stdout.txt

```diff
@@ -35,9 +35,22 @@ Using plugins:
   transform-block-scoping { "ios":"10.3", "safari":"10.1" }
   proposal-export-namespace-from { "android":"61", "chrome":"61", "edge":"16", "firefox":"60", "ios":"10.3", "opera":"48", "safari":"10.1", "samsung":"8.2" }
   syntax-dynamic-import { "android":"61", "chrome":"61", "edge":"16", "firefox":"60", "ios":"10.3", "node":"13.2", "opera":"48", "safari":"10.1", "samsung":"8.2" }
+corejs2: `DEBUG` option
 
-Using polyfills with `usage` option:
+Using targets: {
+  "android": "61",
+  "chrome": "61",
+  "edge": "16",
+  "firefox": "60",
+  "ios": "10.3",
+  "node": "13.2",
+  "opera": "48",
+  "safari": "10.1",
+  "samsung": "8.2"
+}
```

Comment:
The targets are now printed multiple times: once by preset-env (for the transform) and one by the polyfill plugin. I could inject a secret parameter used only by `preset-env` to avoid printing them in the polyfill provider, but I don't think that it's worth the complexity.

Reply:
FWIW, I do think it's worth the complexity :)